content/open-office-hours.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Open Office Hours with Dr. Luis Pedro Coelho"
+meta: "Big Data Biology Lab Office Hours"
+---
+
+[Dr. Luis Pedro Coelho](/people/luis_pedro_coelho) will hold two Zoom open office hour sessions on:
+
+- [April 24, 2024 08:30am AEST (GMT+10.00)](https://qut.zoom.us/meeting/register/tZcsfuCopjwoHdBhmkJQY9mzd0Zu_LYCxrcv#/registration)
```

Comment:
Can you add some info on what the time is in India/Berlin/New York/LA? or a link to the right https://everytimezone.com/ time